WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NotesAdapter.kt

```diff
@@ -0,0 +1,353 @@
+            } else if (timeGroup == NoteTimeGroup.GROUP_OLDER_WEEK) {
+                noteViewHolder.mHeaderText.setText(R.string.older_last_week)
+            } else {
+                noteViewHolder.mHeaderText.setText(R.string.older_month)
+            }
+        }
+
+        // Subject is stored in db as html to preserve text formatting
+        var noteSubjectSpanned: Spanned = note.getFormattedSubject(mNotificationsUtilsWrapper)
+        // Trim the '\n\n' added by HtmlCompat.fromHtml(...)
+        noteSubjectSpanned = noteSubjectSpanned.subSequence(
+            0,
+            TextUtils.getTrimmedLength(noteSubjectSpanned)
+        ) as Spanned
+        val spans = noteSubjectSpanned.getSpans(
+            0,
+            noteSubjectSpanned.length,
+            NoteBlockClickableSpan::class.java
+        )
+        for (span in spans) {
+            span.enableColors(noteViewHolder.mContentView.context)
+        }
+        noteViewHolder.mTxtSubject.text = noteSubjectSpanned
+        val noteSubjectNoticon = note.commentSubjectNoticon
+        if (!TextUtils.isEmpty(noteSubjectNoticon)) {
+            val parent = noteViewHolder.mTxtSubject.parent
+            // Fix position of the subject noticon in the RtL mode
+            if (parent is ViewGroup) {
+                val textDirection = if (BidiFormatter.getInstance()
+                        .isRtl(noteViewHolder.mTxtSubject.text)
+                ) ViewCompat.LAYOUT_DIRECTION_RTL else ViewCompat.LAYOUT_DIRECTION_LTR
+                ViewCompat.setLayoutDirection(parent, textDirection)
+            }
+            // mirror noticon in the rtl mode
+            if (RtlUtils.isRtl(noteViewHolder.itemView.context)) {
+                noteViewHolder.mTxtSubjectNoticon.scaleX = -1f
+            }
+            CommentUtils.indentTextViewFirstLine(noteViewHolder.mTxtSubject, mTextIndentSize)
+            noteViewHolder.mTxtSubjectNoticon.text = noteSubjectNoticon
+            noteViewHolder.mTxtSubjectNoticon.visibility = View.VISIBLE
+        } else {
+            noteViewHolder.mTxtSubjectNoticon.visibility = View.GONE
+        }
+        val noteSnippet = note.commentSubject
+        if (!TextUtils.isEmpty(noteSnippet)) {
+            handleMaxLines(noteViewHolder.mTxtSubject, noteViewHolder.mTxtDetail)
+            noteViewHolder.mTxtDetail.text = noteSnippet
+            noteViewHolder.mTxtDetail.visibility = View.VISIBLE
+        } else {
+            noteViewHolder.mTxtDetail.visibility = View.GONE
+        }
+        val avatarUrl = GravatarUtils.fixGravatarUrl(note.iconURL, mAvatarSz)
+        mImageManager!!.loadIntoCircle(
+            noteViewHolder.mImgAvatar,
+            ImageType.AVATAR_WITH_BACKGROUND,
+            avatarUrl
+        )
+        if (note.isUnread) {
+            noteViewHolder.mUnreadNotificationView.visibility = View.VISIBLE
+        } else {
+            noteViewHolder.mUnreadNotificationView.visibility = View.GONE
+        }
```

Comment:
These lines can be refactored as following code snippet:
```
 noteViewHolder.mUnreadNotificationView.isVisible = note.isUnread
```

Reply:
Resolved via https://github.com/wordpress-mobile/WordPress-Android/pull/20129/commits/d2c57d351a1a99b513e77eb399d357e70d41f861